app/src/main/java/org/p2p/wallet/common/AppSettings.kt

```diff
@@ -0,0 +1,25 @@
+package org.p2p.wallet.common
+
+import android.content.SharedPreferences
+import androidx.core.content.edit
+import org.p2p.wallet.BuildConfig
+
+private const val KEY_POLLING_ENABLED = "KEY_POLLING_ENABLED"
+private const val KEY_IS_PROD = "KEY_IS_PROD"
+
+class AppSettings(private val sharedPreferences: SharedPreferences) {
```

Comment:
It's more like AppFeatureFlags / AppFlags, but not the settings I think

Reply:
it depends of usage
but yes for now I can rename it